wikipron/extract/lat.py

```diff
@@ -0,0 +1,101 @@
+"""Word and pron extraction for Latin.
+
+As of writing (November 2019), Latin cannot use the default extraction
+function because of the following:
+
+1. The default extraction function uses the Wiktionary entry page title
+   as the graphemes. Latin uses the macrons orthographically (for vowel
+   length), but the Wiktionary entry page titles never have them.
+2. Relatedly, because the orthographic distinction by macrons is collapsed,
+   a Latin entry page organizes the "homographs" in terms of "Etymologies".
+   Each etymology has its own (correct) orthographic form and pronunciations.
+
+The solution for Latin is to go through each "Etymology" and extract the
+word and pronunciation at this level.
+"""
```

Comment:
Is the solution (both code and documentation) readable? While it works, I'm also concerned if it'd be too cryptic for our future selves.